egs2/tedlium2/asr1/conf/train_asr_transformer.yaml

```diff
@@ -0,0 +1,62 @@
+batch_type: numel
```

Comment:
I found some differences between this config and that in `conf/tuning`. Which one did you use for the RESULT? We could just keep them in the `tuning` and use soft-link in conf.